Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -44,11 +44,13 @@ extension Mailgun {
 
 extension Database {
   public static let mock = Database(
+    addEpisodeCredit: { _, _ in hole() },
```

Comment:
Hehe. Untested?

Reply:
yikes. ya lots of tests I can write here. will do...